page/login.js

```diff
@@ -8,6 +8,8 @@ function page (req, res) {
   var fs = require('fs')
   fs.readFile('html/login.html', 'utf8', function (err, html) {
     if (err) throw err
-    res.end(require('../src/html_creator').create_html(html, { title: 'Login', js: ['login'] }))
+
+    res.end(require('../src/html_creator').create_html(html, { title: 'Login', js: ['login', 'cookie_parser'] }))
+
```

Comment:
Please use the formatter: `standard --fix`

Reply:
Empty line was accidentally added by me when fixing conflicts in github, fixed by latest commit